packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.test.tsx

```diff
@@ -172,12 +150,15 @@ describe('PageAccountRecoveryConfirmKey', () => {
   describe('submit', () => {
     describe('displays error and does not allow submission', () => {
       it('with an empty recovery key', async () => {
-        render(<Subject {...{ account }} />);
-        fireEvent.click(
-          screen.getByRole('button', { name: 'Confirm account recovery key' })
-        );
+        renderSubject();
+        const submitButton = await screen.findByRole('button', {
```

Comment:
Just curious, why set this to a const that's not reused later in the function (here and in some other functions) instead of leaving what we had before?

Reply:
We need a check to make sure the page content has loaded. In this instance, I can directly await the button element in the fired event (and I'll change that), but for any test where the first action would use `typeByLabelText` we need to `await` another element first (otherwise the test fails).